Makefile

```diff
@@ -51,7 +51,12 @@ check:
 #
 # Example:
 #   make verify
+ifeq ($(SKIP_BUILD), true)
+$(info build is being skipped)
```

Comment:
> Why?

This was the intent of `SKIP_BUILD` all along, but the dependency on `verify` when called transitively through `test` resulted in a `build` regardless of intent